### DISCUSSION TITLE
Quasar PDF Viewer Error

### DISCUSSION THREAD

**Discussion by GaryJurgens** [quasarframework/quasar#10623] [General - Components / Directives / etc]
hi,

I am trying to find or use a pdf viewer for Quasar, 

I found this extension by following the Quazar extension directory. 

[https://quasarframework.github.io/app-extension-qpdfviewer/docs](QpdfViwer)

However, it doesn't seem to want to work, it seems it is not compatible with the latest version Of Quasar or Vue.

Does anyone have any suggestions, options, workarounds? the rest of Quazar is working well.

 App • ⚠️  Extension(@quasar/qpdfviewer): is not compatible with @quasar/app v3.0.1. Required version: ^1.0.0 || ^2.0.0

Thank you for your assistance:

**Comment by hawkeye64** [comment]
@adolfie450 Use the one from the `next` branch.

```
quasar ext add @quasar/qpdfviewer@next
```
Notice the `@next` on the end...

For documentation, go here: https://qpdfviewer.netlify.app/

**Comment by GaryJurgens** [comment]
Hi

Thanks, it works, I think we need to update the quasar docs/list a bit.



Thanks.

**Reply by hawkeye64** [reply]
> Thanks, it works, I think we need to update the quasar docs/list a bit.

Which part of the docs?
